src/coreComponents/physicsSolvers/fluidFlow/StencilAccessors.hpp

```diff
@@ -62,55 +63,65 @@ class StencilAccessors
    * @param[in] solverName the name of the solver creating the view accessors
    */
   StencilAccessors( ElementRegionManager const & elemManager,
-                    string const & solverName )
+                    string const & solverName,
+                    bool const fill = true )
   {
-    forEachArgInTuple( std::tuple< TRAITS ... >{}, [&]( auto t, auto idx )
+    if( fill )
     {
-      GEOSX_UNUSED_VAR( t );
-      using TRAIT = TYPEOFREF( t );
-
-      auto & acc = std::get< idx() >( m_accessors );
-      acc = elemManager.constructExtrinsicAccessor< TRAIT >();
-      acc.setName( solverName + "/accessors/" + TRAIT::key() );
-    } );
+      forEachArgInTuple( std::tuple< TRAITS ... >{}, [&]( auto t, auto idx )
+      {
+        GEOSX_UNUSED_VAR( t );
+        using TRAIT = TYPEOFREF( t );
+
+        auto & acc = std::get< idx() >( m_accessors );
+        acc = elemManager.constructExtrinsicAccessor< TRAIT >();
+        acc.setName( solverName + "/accessors/" + TRAIT::key() );
+      } );
+    }
```

Comment:
What is the role of the `fill` argument that is passed here?

Reply:
Well, when constructing the material version of this `stencilAccessor` the construction of the accessor has to occur differently (need to call `constructExtrinsicMaterialAccessor`) and it's defined in the constructor of the `StencilMaterialAccessor`. However, the constructor of this base class still gets called so I had to add this boolean to avoid building the `ViewAccessor`.

Reply:
Sorry for engaging to late on this... not a super important thing, but I think it would be better to have a `protected` default constructor (which takes no arguments and does not populate accessors) and call that from the derived class, and not use `fill` flags.